### PR TITLE
CLDC-2006 Add and populate local authorities table

### PR DESCRIPTION
**Pull request by kosiakkatrina**
add and populate local authorities table
this will be used for populating answer options for LA questions for different years instead of hardcoding them in the service

### CODE REVIEW

File: app/services/imports/local_authorities_service.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def initialize(path:)
+      @path = path
+      @count = 0
+    end
+
+    def call
+      CSV.foreach(path, headers: true) do |row|
+        LocalAuthority.upsert(
+          { code: row["code"],
+            la_name: row["la_name"],
```
Comment (jamesrose) [inline app/services/imports/local_authorities_service.rb]:
```suggestion
            name: row["la_name"],
```

File: db/schema.rb [hunk 2]
```diff
@@ -289,6 +289,17 @@
     t.index ["updated_by_id"], name: "index_lettings_logs_on_updated_by_id"
   end
 
+  create_table "local_authorities", force: :cascade do |t|
+    t.string "code"
+    t.string "la_name"
```
Comment (jamesrose) [inline db/schema.rb]:
```suggestion
    t.string "name", null: false
```

File: db/schema.rb [hunk 3]
```diff
@@ -289,6 +289,17 @@
...
+    t.datetime "start_date"
```
Comment (jamesrose) [inline db/schema.rb]:
```suggestion
    t.datetime "start_date", null: false
```

File: db/schema.rb [hunk 4]
```diff
@@ -289,6 +289,17 @@
     t.index ["updated_by_id"], name: "index_lettings_logs_on_updated_by_id"
   end
 
+  create_table "local_authorities", force: :cascade do |t|
+    t.string "code"
```
Comment (jamesrose) [inline db/schema.rb]:
```suggestion
    t.string "code", null: false
```

File: app/services/imports/local_authorities_service.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            start_date: Time.zone.local(row["start_year"], 4, 1),
+            end_date: (Time.zone.local(row["end_year"], 3, 31) if row["end_year"]),
+            previous_location_only: row["previous_location_only"] || false },
```
Comment (jamesrose) [inline app/services/imports/local_authorities_service.rb]: What does this field represent?

Comment (kosiakkatrina) [reply app/services/imports/local_authorities_service.rb]: Well, some of the fields are non England as we discussed earlier, so they shouldn't be shown in current la questions but should be shown in previous la questions. I figured marking them in the db would be the easiest way to filter them out when needed

**Review by jamesrose — APPROVED** [review]

**Pull request merged**